yaksh/views.py

```diff
@@ -614,10 +614,43 @@ def add_course(request):
 
 
 @login_required
+def edit_course(request, course_id=None):
```

Comment:
edit_course and add_course are redundant.
For edit course, we can call add_course with course instance.